package.json

```diff
@@ -51,7 +51,7 @@
     "zone.js": "^0.10.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.1102.15",
+    "@angular-devkit/build-angular": "^0.1002.0",
```

Comment:
I don't know a lot about this package, but looking at the [releases page](https://www.npmjs.com/package/@angular-devkit/build-angular), I get the impression that this version is related Angular v10. Do you know if that's the case? If so, I wonder if we should be using it when we're currently on Angular v11.

![Screenshot 2021-12-16 at 09 28 04](https://user-images.githubusercontent.com/1943532/146344767-b2256c8f-8172-4301-a351-4d5f6fb3239d.png)

Reply:
That's what's currently working :/ 11 Hopefully when we upgrade storybook and angular 12 it will all work fine.